packages/shared/src/components/header/BreadCrumbs.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropsWithChildren, ReactElement } from 'react';
+import classed from '../../lib/classed';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { HomeIcon } from '../icons';
+import { WithClassNameProps } from '../utilities';
+
+const BreadCrumbsWrapper = classed(
+  'div',
+  'hidden h-10 gap-0.5 items-center p-1.5 text-border-subtlest-tertiary laptop:flex',
```

Comment:
Wait that's not a text color actually? 👀

Reply:
Oh, looks like it should probably be `text-text-tertiary` 👀 

Will verify this.